x/amm/types/message_exit_pool.go

```diff
@@ -30,6 +31,10 @@ func (msg *MsgExitPool) ValidateBasic() error {
 		}
 	}
 
+	if msg.TokenOutDenom == "" {
```

Comment:
```suggestion
	if msg.TokenOutDenom != "" {
```

Reply:
@amityadav0 We are preventing exiting in single denom, non empty string will allow that

Reply:
right, doesn't this condition throw error when TokenOutDenom is empty ?
it should be throw error when it is not empty ?
am I missing something ?